ml-agents/tests/yamato/yamato_utils.py

```diff
@@ -134,13 +134,10 @@ def init_venv(
     pip_commands = ["--upgrade pip", "--upgrade setuptools"]
     if mlagents_python_version:
         # install from pypi
-        if platform != "darwin":
-            raise RuntimeError("Yamato can only run tensorflow on mac platforms!")
         pip_commands += [
             f"mlagents=={mlagents_python_version}",
             f"gym-unity=={mlagents_python_version}",
             # TODO build these and publish to internal pypi
-            "~/tensorflow_pkg/tensorflow-2.0.0-cp37-cp37m-macosx_10_14_x86_64.whl",
             "tf2onnx==1.6.1",
```

Comment:
This should be removed along with tensorflow package

Reply:
not sure what you mean